File: app/src/main/java/com/noodle/sample/NoodleApp.java
package com.noodle.sample;

import android.app.Application;

import com.noodle.sample.data.BookManager;

public class NoodleApp extends Application {

  private BookManager bookManager;

  public BookManager getBookManager() {
    return bookManager;
  }

  @Override
  public void onCreate() {
    super.onCreate();

    bookManager = new BookManager(this);
  }
}

File: app/src/main/java/com/noodle/sample/NoodleApp.kt
package com.noodle.sample

import android.app.Application

import com.noodle.sample.data.BookManager

class NoodleApp : Application() {

    lateinit var bookManager: BookManager
        private set

    override fun onCreate() {
        super.onCreate()

        bookManager = BookManager(this)
    }
}
